source/conf.py

```diff
@@ -37,7 +37,7 @@
 ]
 
 # Add any paths that contain templates here, relative to this directory.
-templates_path = ['_templates']
+templates_path = ['../_templates']
```

Comment:
Why does this have to change? Should the `_templates` dir be in the same dir as the conf.py?